src/App.js

```diff
@@ -1,12 +1,316 @@
+import Pharagraph from "./Components/Labels/Pharagraph";
+import NameLabel from "./Components/Labels/NameLabel";
+import RadioButton from "./Components/Buttons/RadioButton";
+import CommentLabel from "./Components/Labels/CommentLabel";
+import ConditionsButton from "./Components/Buttons/ConditionsButton";
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    const [strawBerryValue, setStrawberryValue] = React.useState(0)
+    const [bananaValue, setBananaValue] = React.useState(0)
+    const [appleValue, setAppleValue] = React.useState(0)
+    const [kiwiValue, setKiwiValue] = React.useState(0)
+
+    const {register, handleSubmit, formState: {errors, isDirty, isValid}, watch} = useForm({mode: 'onChange'});
+
+    const selectedReferrer = watch("delivery");
+
+    function onFormSubmit(data) {
+        console.log(data)
+    }
+
+    function strawBerryValueMinusOne() {
+        if (strawBerryValue === 0) {
+            return false
+        } else {
+            setStrawberryValue(strawBerryValue - 1)
+        }
+    }
+
+    function strawBerryValuePlusOne() {
+        setStrawberryValue(strawBerryValue + 1)
+    }
+
+    function bananaValueMinusOne() {
+        if (bananaValue === 0) {
+            return false
+        } else {
+            setBananaValue(bananaValue - 1)
+        }
+    }
+
+    function bananaValuePlusOne() {
+        setBananaValue(bananaValue + 1)
+    }
+
+    function appleValueMinusOne() {
+        if (appleValue === 0) {
+            return false
+        } else {
+            setAppleValue(appleValue - 1)
+        }
+    }
+
+    function applePlusOne() {
+        setAppleValue(appleValue + 1)
+    }
+
+    function kiwiValueMinusOne() {
+        if (kiwiValue === 0) {
+            return false
+        } else {
+            setKiwiValue(kiwiValue - 1)
+        }
+    }
+
+    function kiwiPlusOne() {
+        setKiwiValue(kiwiValue + 1)
+    }
+
+    function resetValue() {
+        setStrawberryValue(0)
+        setBananaValue(0)
+        setAppleValue(0)
+        setKiwiValue(0)
+    }
+
+
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+
+            <form onSubmit={handleSubmit(onFormSubmit)}>
+
+                <div className="fruit">
+                    <h2>🍓 Aardbeien</h2>
+                    <PlusButton
+                        className="button"
+                        plusOne={strawBerryValuePlusOne}>
+                        +
+                    </PlusButton>
+
+                    <Pharagraph
+                        fruitName="strawberries"
+                        fruitValue={strawBerryValue}
+                        register={register}
+                    >{strawBerryValue}</Pharagraph>
+
+                    <Minusbutton
+                        className="button"
+                        minusOne={strawBerryValueMinusOne}>
+                        -
+                    </Minusbutton>
+
+                </div>
```

Comment:
Hier had je een fruitcomponent van kunnen maken zodat je niet zoveel code hoeft te dubbelen. Dan had ik de set-functies meegegeven die uit de setState's komen en in het component zelf de is-groter-dan-nul-dan-minus-functie ingezet